package.json

```diff
@@ -106,6 +106,7 @@
     "react": "^16.13.0",
     "react-animate-height": "^2.0.20",
     "react-dom": "^16.13.0",
+    "react-router-dom": "^5.1.2",
```

Comment:
I don't see anything about routing in the spec

Reply:
Lies

https://github.com/rgsystemes/lib-ui/blob/4ddf236b4f982dd9583bc9fcab5b4ff08d3cf3df/src/Molecules/Header/Header.jsx#L2

Reply:
Nevermind, I misread your comment. It had to be added in order to be able to `link` between componants using a router.

Reply:
![image](https://user-images.githubusercontent.com/717869/80081113-cc529c80-8552-11ea-8dc6-b724fdf85b2f.png)